test/Container/ApplicationFactoryTest.php

```diff
@@ -1071,4 +1071,226 @@ public function testCanSpecifyRouteOptionsViaConfiguration()
         $this->assertInstanceOf(Route::class, $route);
         $this->assertEquals($expected, $route->getOptions());
     }
+
+    /**
+     * @expectedException \Zend\Expressive\Container\Exception\InvalidArgumentException
```

Comment:
My rule is:

1. It must be unique for don't be confused with any other exception of the same type
2. Make the less efort (usually copy the whole message), if too much then only the essential part for acomplish the first rule.